public-lighting-demo-app/web-demo-app/pom.xml

```diff
@@ -12,26 +12,25 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>web-demo-app</artifactId>
```

Comment:
I wouldn't expect changes in this file

Reply:
Pom changes are result of pom reordering according to pom code conventions to fix sonar warning and switching to openjdk 11 on local environment. 
Would have been better to create separate PR for this...